pipelines/environments/mldesigner-2.0-ubuntu20.04-py38-cpu/asset.yaml

```
@@ -0,0 +1,5 @@
+name: mldesigner-2.0-ubuntu20.04-py38-cpu
```

Would `mldesigner` work for you as the environment name, if the `2.0` designation isn't important? We're trying to lean toward simpler environment names where possible. See https://github.com/Azure/azureml-assets/wiki/Asset-naming-convention#environments
```suggestion
name: mldesigner
```